src/config.ts

```diff
@@ -1376,6 +1382,47 @@ export const EmbedChainInfos: ChainInfoWithExplorer[] = [
 		features: ['stargate', 'ibc-transfer', 'no-legacy-stdTx', 'ibc-go'],
 		explorerUrlToTx: 'http://blockexplorer-1.testnet.saage.io/transactions/{txHash}',
 	},
+
+	{
+		rpc: 'https://tx-endpoint-test.imversed.com:443',
+		rest: 'https://query-endpoint-test.imversed.com:443',
+		chainId: 'imversed-test-1',
+		chainName: 'imversedtestnet',
+		stakeCurrency: {
+			coinDenom: 'IMV',
+			coinMinimalDenom: 'nimv',
+			coinDecimals: 6,
+			coinImageUrl: window.location.origin + '/public/assets/tokens/imversed.png',
+			coinGeckoId: 'imversedtest',
+		},
+		bip44: { coinType: 118 },
+		bech32Config: Bech32Address.defaultBech32Config('imv'),
+		currencies: [
+			{
+				coinDenom: 'IMV',
+				coinMinimalDenom: 'nimv',
+				coinDecimals: 6,
+				coinImageUrl: window.location.origin + '/public/assets/tokens/imversed.png',
+				coinGeckoId: 'imversedtest',
```

Comment:
is 'imversedtest' really a CoinGecko ID? I couldn't find it in here: https://api.coingecko.com/api/v3/coins/list

Reply:
Can I just comment coinGeckoId param?